crates/telio-nurse/src/qos.rs

```diff
@@ -416,12 +460,18 @@ mod tests {
 
         let expected_output = OutputData {
             rtt: String::from("20:40:70:90:100,0:0:0:0:0,20:40:70:90:100"),
+            rtt_loss: String::from("20:40:70:90:100,0:0:0:0:0,20:40:70:90:100"),
```

Comment:
Not sure what was agreed, but should'nt `loss` be represented as just simple percentage number? Or event attempts vs success counts (this way we will know if we get 50% because we lost one packet, but attempted only twice, or whether we genuinely have 50% loss.

Reply:
Yup, but my impression here was, that these buckets are formed from individual ping sessions (it represents the loss percentage `20%:40%:70%:90%:100%`).